.github/workflows/deploy_module.yml

```diff
@@ -72,7 +72,7 @@ jobs:
         name: appimage-${{ matrix.platform }}
     - id: stripped-version
       run: echo version=${VERSION#v} >> $GITHUB_OUTPUT
-    - uses: viamrobotics/upload-module@main
+    - uses: viam-modules/upload-module@main
```

Comment:
i think this change needs to be reverted? or at least I didn't change this for cartographer & my module uploaded

Reply:
done! going to merge now